### PR TITLE
Add documentation for catalog subscriptions

### PR DESCRIPTION
**Pull request by rsavajiyani**
This documentation update includes information about how to manage catalog item subscriptions. I also created placeholder documentation for the AMPLIFY Catalog, using the text that was already present in the overview.md file. This will need to be iterated on and improved in the future.

@alexearnshaw @andreamussap  this is my first time contributing to the documentation, so I'd appreciate any feedback you'd have. I wasn't sure what the "weight" property in the table at the top of the documentation markdown files was.

@robertengaxway please let me know if this is sufficient. I tried to make the language I used consistent with the language used in the Catalog videos on Axway's youtube channel, such as https://youtu.be/_87-k4bxJQ4

I'm in the process of creating GIFs to embed into this documentation that steps through the subscription process.

### CODE REVIEW

File: content/en/docs/central/catalog_overview.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+linkTitle: Overview
+weight: 1
+date: 2019-12-16
+description: Learn what the AMPLIFY Catalog is
+---
+
+*Estimated reading time: 1 minute*
+
+## What is the AMPLIFY Catalog?
+The AMPLIFY Catalog provides a registry of all endpoints belonging to an organization. Examples of endpoints are APIs, MFT flows, SOAP, or gRPC, and they can be in different environments such as in the cloud, on-premise, within a microservice mesh, or at the edge (DMZ) of an organization. AMPLIFY Catalog allows developers to quickly discover and understand the endpoints and protocols that they can use to integrate quickly with your services. It simplifies access and increases the speed of building, integrating and deploying new services and apps.  Platform administrators can limit content access to specific users or teams and [manage subscriptions](/docs/central/catalog_subscriptions).
```
Comment (georgiana-caras) [inline content/en/docs/central/catalog_overview.md]: I suggest replacing the Overview with the following text.  "The AMPLIFY Catalog provides a common view of all endpoints belonging to an organization. Alongside API interfaces, you'll have Managed File Transfer, business-to-business connectors that move data across your enterprise and outside its boundaries, and they can be in different environments such as in the cloud, on-premise , within a microservice mesh, or at the edge (DMZ) of the organization. AMPLIFY Catalog allows developers to quickly discover and understand the endpoints and protocols that they can use to integrate quickly with your services. It simplifies access and increases the speed of building, integrating and deploying new services and apps.  Platform administrators can limit content access to specific users or teams and [manage subscriptions](/docs/central/catalog_subscriptions)."

**Comment by andreamussap** [comment]
@rsavajiyani well done for trying out the doc-as-code process. Feel free to ask for any question or help that you need.

The weight specifies the position of this topic in the [AMPLIFY Central documentation menu](https://axway-open-docs.netlify.com/docs/central/). I can update this later. I just need @georgiana-caras  to confirm that the new topic 'Catalog Overview' can be placed at the bottom of the menu, just after 'Configure a CFT Profile' ?? 

As soon as you apply Georgiana's feedback, I'll perform the editorial review.

File: content/en/docs/central/catalog_overview.md (hunk 1 above)
Comment (rsavajiyani) [reply content/en/docs/central/catalog_overview.md]: Thanks @georgiana-caras , I changed it to use your wording.

**Comment by andreamussap** [comment]
@rsavajiyani I've performed the editorial review:

**catalog_overview.md**  -->  removed because it's just a paragraph!
**catalog_subscriptions.md**  -->  Topic was renamed and 'Overview' paragraph was moved into here. 
**catalog_delete_subscription.gif** --> removed because it's very similar to the catalog_unsubscribe.gif

* The resolution of the gifs is poor. They're not helpful if you barely can see what's happening on the screen. Can you recreated them with better resolution?
* I've added a description (right before the title of the page) and I tweaked the overview paragraph and, although I'm not happy with the text, I cannot do more because I don't have knowledge for that, so I need you or in fact, @georgiana-caras  to improve and clarify the text of the description and overview sections. 

Here is the directly link to the preview of the topic - https://deploy-preview-200--axway-open-docs.netlify.com/docs/central/catalog/

**Comment by andreamussap** [comment]
@georgiana-caras  , for now there's only one topic/page for the Catalog docs but I'd like to know what's the plan for this documentation? How many features, concepts, procedures do you think will have more to add to the documentation?

**Comment by georgiana-caras** [comment]
@andreamussap , I would imagine having sections for explaining how to edit a catalog item, how to edit the overview markdown documentation, share a catalog item with a team, explain how to find an asset. There is future backlog to support categories and the management of that, improved filtering capabilities by short description and content in the swagger file. All of that needs to have separate sections in the documentation.

**Comment by andreamussap** [comment]
RE the focus is only on APIs --> The reason for that is that I got that text for some Jive page where you were talking about the Catalog, just to make the content more interesting, but as I said before, I don't have the required understanding to write the correct information covering what the Catalog really encompass. 

RE paragraphs explaining how to subscribe to an api vs to an mft service --> it makes sense. Since you said you're happy with the content I'll approve and merge it, once the page is added to the documentation it will be easier for you/your team to keep adding more and improving the Catalog doc by simply creating new pull requests like this. Thank you. /Andrea

**Comment by andreamussap** [comment]
The first page for the Catalog is released in the documentation now - https://axway-open-docs.netlify.com/docs/central/catalog/

Any changes to this content must be done via a new PR.

**Comment by andreamussap** [comment]
### Thank you
We really appreciate your contribution and thank you for helping us to improve this documentation!
 
### Where to find the updated documentation
An Axway technical writer has reviewed and merged your PR, which is now available on the following sites:
Netlify (staging): https://axway-open-docs.netlify.com/docs/
Zoomin (production): https://docs.axway.com/bundle/axway-open-docs/
 
### Send us your feedback
If you have a moment, please complete the following 2 minute survey to give us your feedback on this process and help us make it even better:
https://www.surveymonkey.com/r/PJ5HQFD

**Pull request merged**